### ISSUE TITLE
readonly properties with template

### ISSUE THREAD

**Issue by bennypowers** [web-padawan/api-viewer-element#15]
I have a component with some read-only properties defined as getters.

When I use a template to set up the initial demo state, I get errors about trying to set those read only props.

```html
<script src="https://unpkg.com/shaka-player"></script>
<script src="https://unpkg.com/api-viewer-element?module" type="module"></script>
<script type="module">
  import './shaka-player.js'; //load the component
</script>
<api-viewer src="./custom-elements.json">
  <template data-element="shaka-player" data-target="host">
    <shaka-player controls dash-manifest="http://amssamples.streaming.mediaservices.windows.net/683f7e47-bd83-4427-b0a3-26a6c4547782/BigBuckBunny.ism/manifest(format=mpd-time-csf)"></shaka-player>
  </template>
</api-viewer>
```

Switching to Demo tab gives: 
```
TypeError: setting getter-only property "duration"
```

**Comment by web-padawan** [comment]
Hi, there is currently a check that attempts to discover getters here:

https://github.com/web-padawan/api-viewer-element/blob/4537cc886d107ad6954e177061cdeff9e520733a/src/api-viewer-demo-layout.ts#L52-L69

Maybe it isn't working for your element. I will debug to investigate.

**Comment by bennypowers** [comment]
relevant to https://github.com/bennypowers/shaka-player/tree/7dcdd1410b0f258f64afca8eb72e3cc50208ea0b

**Comment by web-padawan** [comment]
Fixed by fc2171d and released as `0.3.13`. I tested the new version and now it no longer throws.

From now on, readonly properties are excluded from the demo page completely.
Neither they appear in knobs, nor the default values for them are initialised.

Thanks for reporting and sorry about the 3 weeks delay!
